src/common/pf_ppm.h

```diff
@@ -117,6 +120,9 @@ int pf_ppm_get_data_and_iops (
 
 /**
  * Retrieve IOCS for a sub-module.
+ *
+ * This is for reading back the IOCS you have set to the PPM.
```

Comment:
Repetition?

Reply:
Changed to 
`Note that this is not from the PLC, but previously set by the application.`